Lavinia-api/Utilities/CsvFormats.cs

```diff
@@ -113,8 +113,8 @@ public ResultFormat Parse(string line, FieldParser parser)
     /// </summary>
     public class CountryFormat : ICsvFormat<CountryFormat>
     {
-        public string CountryCode;
```

Comment:
If I recall correctly these properties are set in constructor only -- it is then more appropriate to use a public readonly get and have no setter to ensure no one thinks it is OK to use the private setters in the method.